.gitmodules

```diff
@@ -91,8 +91,8 @@ fxDONOTUSEurl = https://github.com/ESCOMP/CMEPS.git
 
 [submodule "cdeps"]
 path = components/cdeps
-url = https://github.com/ESCOMP/CDEPS.git
-fxtag = cdeps1.0.75
+url = https://github.com/samsrabin/CDEPS.git
+fxtag = 304d112e23019579013d08e57a463334ac8cac8f  # cx-length-filenames: Use CX-length filenames for all *_comp_nuopc.F90.
```

Comment:
@samsrabin we didn't discuss this in our paired code-review. Do you plan to:
- [ ] merge your cdeps PR to cdeps main FIRST?
- [ ] then update .gitmodules before merging this ctsm PR to b4b-dev?

That would be my preference, unless you can justify why you should not do things in that order.

Reply:
Oh yes, this will all have to wait on the CDEPS PR (ESCOMP/CDEPS#343).

@ekluzek, could I get that fast-tracked?